src/lib/components/home/artwork_rails/artwork_rail.tsx

```diff
@@ -207,13 +207,21 @@ class ArtworkRail extends React.Component<Props & RelayPropsWorkaround, State> {
     if (this.state.loadFailed) {
       return null
     }
+    const hasStartedFetching = this.props.relay && this.props.relay.variables.fetchContent
 
     const sideMargin = isPad ? 40 : 20
     const style: any = { marginLeft: sideMargin, marginRight: sideMargin }
-    if (!(this.props.relay && this.props.relay.variables.fetchContent)) {
+    if (!hasStartedFetching) {
       // if there's no content, set minHeight to prevent spinners from all the rails showing
       style.minHeight = 400
```

Comment:
This doesn’t look right. The rails should always start at min 400, currently they only show part of the spinner for a brief period of time.